src/uct/tcp/tcp_iface.c

```diff
@@ -80,6 +80,10 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
   UCT_IFACE_MPOOL_CONFIG_FIELDS("RX_", -1, 8, "receive",
                                 ucs_offsetof(uct_tcp_iface_config_t, rx_mpool), ""),
 
+  {"PORTS_RANGE", "0",
+   "Range of ports to use. Zero means to use a random port.",
```

Comment:
Generate a random TCP port number from that range. A value of zero means let the operating system select the port number.